src/sdkV2.ts

```diff
@@ -490,4 +522,14 @@ export default class CegaEvmSDKV2 {
     const oracleEntry = await this.loadOracleEntry();
     return oracleEntry.getPrice(baseAsset, quoteAsset, timestamp, oracleDataSource);
   }
+
+  async pythUpdateAssetPrices(
+    timestamp: number,
```

Comment:
this should be a `Date`, and in the function you can convert it into the corresponding `number`

the SDK *caller* shouldn't have to be concerned about 1701052220 vs 1701032220. but rather `"Monday 2pm UTC" etc.